databricks-sdk-java/src/main/java/com/databricks/sdk/core/AzureCliCredentialsProvider.java

```diff
@@ -37,12 +37,24 @@ public HeaderFactory configure(DatabricksConfig config) {
       ensureHostPresent(config, mapper);
       String resource = config.getEffectiveAzureLoginAppId();
       CliTokenSource tokenSource = tokenSourceFor(config, resource);
+      CliTokenSource mgmtTokenSource =
+          tokenSourceFor(config, config.getAzureEnvironment().getServiceManagementEndpoint());
       tokenSource.getToken(); // We need this for checking if Azure CLI is installed.
+      try {
+        mgmtTokenSource.getToken();
+      } catch (Exception e) {
+        LOG.debug("Not including service management token in headers", e);
+        mgmtTokenSource = null;
+      }
+      CliTokenSource finalMgmtTokenSource = mgmtTokenSource;
```

Comment:
What is final about it? Seems like the alias is not necessary below.

Reply:
Java style doesn't allow capturing non-final variables in closures. https://www.baeldung.com/java-lambda-effectively-final-local-variables#capturing-lambdas